device/net/Kconfig

```diff
@@ -0,0 +1,6 @@
+#
+# SCM Network Support
+#
+
+config HSS
```

Comment:
Maybe `config HSS_SOCKET` to differentiate it from the `config HSS` used for the host driver.

Reply:
That works.